### PR TITLE
Sweeping fixes to va-icon and buttons that use icons

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://9999-va-icon-sizing-fix--65a6e2ed2314f7b8f98609d8.chromatic.com

---

## Description
Fixes a va-icon alignment issue generally and in buttons that use icons specifically.

## QA Checklist
- [X] Component maintains 1:1 parity with design mocks
- [X] Text is consistent with what's been provided in the mocks
- [X] Component behaves as expected across breakpoints
- [X] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [X] Tab order and focus state work as expected
- [X] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [X] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [X] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Screenshots
Before: 
image

After: Screenshot 2024-04-30 at 4 18 23 PM

## Acceptance criteria
- [ ] QA checklist has been completed
- [X] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/src/components/va-button/va-button.scss [hunk 1]
```diff
@@ -17,22 +17,13 @@
 }
 
 // Fix to off-centered content in V3 when icon present
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-button/va-button.scss:19]: I see some CSS deleted. Do you need this comment here still?

Comment (Andrew565) [reply packages/web-components/src/components/va-button/va-button.scss:19]: Yes, that comment still applies to the remaining CSS.

**Review by jamigibbs — APPROVED** [review]

**Review by jamigibbs — COMMENTED** [review]
Sorry, before I approve, do you mind confirming the visuals for this? The icons on button pair look different to me for some reason. Maybe smaller?


**before**

https://design.va.gov/storybook/?path=/docs/uswds-va-button-pair--default

Screenshot 2024-04-30 at 2 20 47 PM


**after**

https://9999-va-icon-sizing-fix--65a6e2ed2314f7b8f98609d8.chromatic.com/?path=/docs/uswds-va-button-pair--continue

Screenshot 2024-04-30 at 2 21 52 PM

**Review by micahchiang — COMMENTED** [review]
These changes look okay to me, but it'd be great if screenshots could be provided that verify these fixes address the issues reported in vets-website

**Comment by Andrew565** [comment]
Yes @jamigibbs the continue and back button chevrons are smaller. @danbrady will need to verify that that's OK.

**Comment by danbrady** [comment]
Material icons are a bit smaller than FA even when sized the same. They also have thinner strokes/lines which accentuates the difference. Here's the difference in Figma:

Here's how the buttons looked with FA:
image

Here's how they look swapped with Material/USWDS icons, keeping the same size/spacing:
image

Granted, with the smaller icons, the spacing looks like *too* much.

However, it looks like somewhere along the way we lost most of the spacing between the text and the icon in these buttons.

So I think we might need to tweak the spacing some. I'll adjust Figma and @Andrew565 , I'll connect with you on the right design.

**Comment by Andrew565** [comment]
> I'll adjust Figma and @Andrew565 , I'll connect with you on the right design.

Thanks Dan!

**Comment by danbrady** [comment]
@Andrew565 Ok, did some quick specs. Because the icon is in the button, it should get it's size from the font. Here's some quick specs of the ideal layout:

image

Here's how it looks as in the group:

image

I did the same thing using USWDS code for buttons and icons, and you get the same layout! (So that tells me we're on the right path here.) Screenshot below:
image

**Comment by danbrady** [comment]
Also, if a designer _does_ put an icon in a large variation of the button, it should follow the same specs. In other words, the icon size should match the bigger text size with 8px spacing between.

**Comment by Andrew565** [comment]
Spacing adjusted, @danbrady.
va-button-icon:
Screenshot 2024-05-01 at 11 20 20 AM
va-button - Continue and Back:
Screenshot 2024-05-01 at 11 21 25 AM
va-button - Big with back (for demonstration purposes):
Screenshot 2024-05-01 at 11 22 24 AM

**Comment by danbrady** [comment]
LGTM!

**Review by micahchiang — APPROVED** [review]
Looks good. Thank you for the quick turn around @Andrew565

**Pull request merged**